packages/core/src/connectivity/scp-cf/proxy-util.ts

```diff
@@ -266,18 +270,21 @@ export function proxyAgent(
     throw new Error('Proxy config must not be undefined.');
   }
 
+  const agentConfig = {
+    host: proxyConfig.host,
+    protocol: proxyConfig.protocol,
+    port: proxyConfig.port,
+    ...options
```

Comment:
Should we add a warning if `options` contain some keys like `host` to make our life easy when debugging?

Reply:
good idea. I do that.